packages/apsara-ui/src/Segments/Segments.tsx

```diff
@@ -1,16 +1,19 @@
 /* eslint-disable react/no-array-index-key */
-import React from "react";
-import { Col, Collapse } from "antd";
-import { Row, Key, Value, Title, Wrapper } from "./Segments.styles";
-
-const { Panel } = Collapse;
+import React, { useState } from "react";
+import { Row, Key, Value, Title, Wrapper, CollapsibleHeader } from "./Segments.styles";
+import * as Collapsible from '@radix-ui/react-collapsible';
+import { ChevronRightIcon, ChevronDownIcon } from "@radix-ui/react-icons";
 
 interface SegmentsProps {
     span?: number;
     children?: React.ReactNode;
 }
 
-const Segments = ({ span = 10, children }: SegmentsProps) => <Col span={span}>{children}</Col>;
+const calculateWidth = (span: number) => {
+    return (span / 24) * 100 + '%'
+}
+
+const Segments = ({ span = 10, children }: SegmentsProps) => <div style={{ maxWidth: calculateWidth(span) }}> {children}</ div>;
```

Comment:
do we really need span here? or is it just internal use?

I feel like we can skip `span`, CMIIW it was there because we use `Col` as the internal component for `Segment`?

Reply:
I retained span here just to preserve the backward compatibility, 
with `Col` being used previously span was being used there as well just to set the width.